### DISCUSSION TITLE
java: package io.swagger.client.api does not exist

### DISCUSSION THREAD

**Discussion by zzhou37** [amzn/selling-partner-api-models#3488] [General]
I followed all instruction at the page https://developer-docs.amazon.com/sp-api-blog/docs/automate-your-sp-api-calls-using-java-sdk, however after I import all the models in lntelliJ there is one package can't find. I checked all the Java SDK generated, and I do find the package is right there.
What problem it might be?

**Comment by puppsupr** [comment]
Hi zzhou37@,

What is the missing package ?

Thanks and Regards,
Supriya P
Solutions Architect, Selling Partner API
